zerver/lib/export.py

```diff
@@ -2594,3 +2604,33 @@ def get_realm_exports_serialized(realm: Realm) -> list[dict[str, Any]]:
             export_type=export.type,
         )
     return sorted(exports_dict.values(), key=lambda export_dict: export_dict["id"])
+
+
+def get_migrations_by_app() -> AppMigrations:
+    from django.db import DEFAULT_DB_ALIAS, connections
+    from django.db.migrations.recorder import MigrationRecorder
+
+    recorder = MigrationRecorder(connections[DEFAULT_DB_ALIAS])
```

Comment:
This is the default setting `connections` parameter we use to call `get_migration_status` which is likely calls to the same `applied_migrations()` function I use here.